win32ss/user/winsrv/consrv/condrv/text.c

```diff
@@ -1042,6 +1169,18 @@ ConDrvFillConsoleOutput(IN PCONSOLE Console,
                 Y = 0;
             }
         }
+
+        bLead = !bLead;
+    }
+
+    if ((NumCodesToWrite & 1) & bFullwidth)
+    {
+        if (X + Y * Buffer->ScreenBufferSize.X > 0)
+        {
+            Ptr = &Buffer->Buffer[X + Y * Buffer->ScreenBufferSize.X - 1];
```

Comment:
Shouldn't we use `Ptr = ConioCoordToPointer(Buff, X - 1, Y);` instead ?

Reply:
What is `VirtualY`?

Reply:
See this comment: https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/winsrv/consrv/include/conio.h;hb=6ad4f9e7bbab0effbdb18423f842dc6ab1d1dc12#l125

Reply:
OK.